scripts/parse_syscalls.py

```diff
@@ -125,7 +126,7 @@ def parse_args():
         description=__doc__,
         formatter_class=argparse.RawDescriptionHelpFormatter)
 
-    parser.add_argument("-i", "--include", required=True,
+    parser.add_argument("-i", "--include", required=True, nargs="*",
```

Comment:
Instead of passing `-i <list of directories>`, can we instead support multiple -i calls which build up a list? Sort of like how with GCC you pass -I multiple times for each include path.

Reply:
Sure, should be pretty straight fwd. let it put it up today.